instruments/nas/instuments.go

```diff
@@ -11,3 +11,13 @@ type CreateCustomerInstrumentRequest struct {
 	Phone   *common.Phone `json:"phone,omitempty"`
 	Default bool          `json:"nas,omitempty"`
 }
+
+type InstrumentSectionFieldAllowedOption struct {
+	Id      string `json:"id,omitempty"`
+	Display string `json:"display,omitempty"`
+}
+
+type InstrumentSectionFieldDependencies struct {
+	FieldId string `json:"field_id,omitempty"`
+	Value   string `json:"value,omitempty"`
+}
```

Comment:
Maybe this ones should be relocated to `get.go` as they are only used in `InstrumentSectionField` struct

Reply:
👍